modules/pool/invest/components/PoolInvestCustom.tsx

```diff
@@ -33,9 +33,12 @@ export function PoolInvestCustom({ onShowPreview }: Props) {
                     <PoolInvestSummary />
                 </Box>
                 {pool.investConfig.options.map((option, index) => {
+                    const tokenOptions = option.tokenOptions.filter(
```

Comment:
Would not remove this as an option, you then make discoverability much harder

Reply:
Commit reverted.